src/main/resources/assets/src/customerView.js

```diff
@@ -94,6 +94,7 @@ class CustomerView extends React.Component {
     render() {
         return (
             <div>
+                <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
```

Comment:
We already have this code in the [html file](https://github.com/jerometujague/SoftwareEngineeringProject/blob/fb60a704357e2633df21ad0544f7f4d477917470/src/main/resources/assets/src/index.html#L9). Does putting it here change anything?